package.json

```diff
@@ -1,4 +1,8 @@
-{
+{"jest": {
+  "snapshotSerializers": ["enzyme-to-json/serializer"],
+  "collectCoverageFrom": ["src/**/*.js", "!src/index.js"],
+  "coverageReporters": ["text"]
```

Comment:
This is not required. Instead add code coverage constraint. Find out on google on how to add global code coverage threshold on jest.
```suggestion
```